cli/Cargo.toml

```diff
@@ -55,8 +55,7 @@ source-map-mappings = "0.5.0"
 sys-info = "0.5.8"
 tempfile = "3.1.0"
 termcolor = "1.0.5"
-tokio = { version = "0.2.9", features = ["full"] }
-tokio-executor = "0.1.9"
+tokio = { version = "0.2", features = ["full"] }
```

Comment:
why not pinning minor?

Reply:
because I think tokio is backwards compatible on minor?

Reply:
Upgrade `0.2.6` -> `0.2.9` had breaking changes:
https://github.com/denoland/deno/pull/3653/files